pubspec.lock

```diff
@@ -7,7 +7,7 @@ packages:
       name: analyzer
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.32.4"
+    version: "0.31.2-alpha.2"
```

Comment:
@natpenguin `pubspec.lock` seems to be changed. Would you update flutter version?